Helpers.gs

```diff
@@ -355,10 +355,14 @@ function createEvent(event, calendarTz){
   if (event.hasProperty('location'))
     newEvent.location = icalEvent.location;
 
-  if (event.hasProperty('class')){
-    var classString = event.getFirstPropertyValue('class').toString().toLowerCase();
-    if (["default", "public", "private", "confidential"].indexOf(classString) > -1)
-      newEvent.visibility = classString;
+  if ( ["default", "public", "private"].includes(forceVisibility.toLowerCase()) ) {
+    newEvent.visibility = forceVisibility;
+  } else {
+    if (event.hasProperty('class')){
```

Comment:
This `if` can be combined with the `else` above (ie `else if`). No need to have them separate.

Reply:
agreed and done